truss/truss/config.go

```diff
@@ -0,0 +1,41 @@
+package truss
+
+import (
+	"path/filepath"
+)
+
+// Config defines the inputs to a truss service generation
+type Config struct {
+	// The first path in $GOPATH
+	GOPATH string
+
+	// The path to where .pb.go files protoc-gen-go creates will be written
```

Comment:
Could be worthwhile to add to the comment above the config struct itself instead to mention/point to where that these values have defaults exposed by the command arguments. I'm not sure theres a worthwhile comment that wouldn't go stale though. 

I'm inclined to agree with adam on the comment.

Separately is this a filepath or a go package? Name feels like the former, though we had been discussing the later. If it is the later we should probably rename to something `PBPackage`, `PBOutPackage`